yarn-project/sequencer-client/src/config.ts

```diff
@@ -88,5 +90,9 @@ export function getConfigEnvVars(): SequencerClientConfig {
     feeRecipient: FEE_RECIPIENT ? AztecAddress.fromString(FEE_RECIPIENT) : undefined,
     acvmWorkingDirectory: ACVM_WORKING_DIRECTORY ? ACVM_WORKING_DIRECTORY : undefined,
     acvmBinaryPath: ACVM_BINARY_PATH ? ACVM_BINARY_PATH : undefined,
+    allowedFeePaymentContractClasses: SEQ_FPC_CLASSES ? SEQ_FPC_CLASSES.split(',').map(Fr.fromString) : [],
```

Comment:
Can you just check that this works from the command line? When setting up a node in the `aztec` package.

Reply:
The new env vars works correctly but the `aztec start` sequencer options (e.g. `aztec start --sequencer allowedFeePaymentContractClasses=0xabcdef,feeRecipient=0x12345`) are not parsed at all so the sequencer gets them as strings instead of addresses/arrays.

This will require a bit of refactoring, I'll open a separate PR with a fix to parse all of the sequenecer's config options.